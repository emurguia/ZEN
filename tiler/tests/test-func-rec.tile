// testing that recursive programmer-defined functions work

int fib(int x) {
	if (x <= 1) {
		return x;
	}
	return fib(x-1) + fib(x-2);
}

init {
	tile(3,3);
	iprint(fib(5));
	close();
}
