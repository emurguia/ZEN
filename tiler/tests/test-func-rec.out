5
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.
